.github/workflows/tests.yml

```diff
@@ -0,0 +1,185 @@
+name: Tests
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: master
```

Comment:
Should we also run these for the `0.1` branch?

Reply:
The `0.1` branch will require a slightly different workflow (e.g. due to the differences in features), which I think can live separately in that branch.

Reply:
Sounds good to just have a separate config in that branch.